composer.json

```diff
@@ -25,7 +25,7 @@
         "ext-json":                        "*",
         "dflydev/fig-cookies":             "^2.0.1",
         "lcobucci/jwt":                    "^3.3.2",
-        "lcobucci/clock":                  "^1.3.0",
+        "lcobucci/clock":                  "^2.0.0",
```

Comment:
This PR **is** backward compatible, but since you're Marco Pivitti, let's shoot down the past, right?

Reply:
But but but I did my best to break BC on the upstream dependency :cry: 

Where's the fun, @Slamdunk?? :rofl: